packages/api/src/libs/queries/lsp.ts

```diff
@@ -87,15 +95,52 @@ export default (appState: Dependencies) => {
     const result = await getTotalTvlSample(currency);
     return result.value;
   }
+  // tvm calculations, these are just based on tvl
+  async function sumTvm(addresses: string[], currency: CurrencySymbol = "usd") {
+    const tvms = await Promise.all(
+      addresses.map(async (address) => {
+        try {
+          const stat = await getTvm(address, currency);
+          return stat.value || "0";
+        } catch (err) {
+          return "0";
+        }
```

Comment:
Similar to previous comments, why do we need to bury errors here? Is this expected in the happy path? If so, is there a different logic we could use to detect empty? If it shouldn't happen, should we be printing an error message so we can see it for debugging purposes?

Reply:
This is really built for the frontend to call, IMO i dont want these calls to throw errors unless an invalid address is specified.  There are all kinds of errors on the ingestion side which can lead to no TVM/TVL for a contract, these are logged with the contract address on ingestion side.  If we passed these errors to the user on every query nothing would work. But we still have the ability to debug if the numbers seem wrong by searching through ingestion logs. See previous comment for example logs.